serializer/serializer.h

```diff
@@ -12,6 +12,12 @@
 extern "C" {
 #endif
 
+#define DEFAULT_TAG "AAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+#define DEFAULT_ADDRESS              \
+  "JFTON9DYPETJJ9BGBJMJQPCIXEPPGIGH" \
+  "IEVHPFHZQ9PMZFYOEPCO9TPPJZZTJBHNRFVDAOJZSY9GQEJIZ"
+#define DEFAULT_MSG "EMPTYMESSAGE"
```

Comment:
I would like to set default strings to following:
* tag: "POWEREDBYTANGLEACCELERATOR9"
* address: "POWEREDBYTANGLEACCELERATOR999..."
* message: "ZBCDKDTCFDTCSCEAQCMDEAHDPCBDVC9DTCRAPCRCRCTC9DTCFDPCHDCDFD" (which is "Powered by tangle-accelerator" in ascii)